metagov/metagov/plugins/github/models.py

```diff
@@ -113,16 +113,28 @@ def method(self, parameters):
         except PluginErrorInternal as e:
             logger.warn(f"Method {interpolated_route} failed with error {e}")
 
+    @Registry.action(
+        slug='create-issue',
+        description='creates issue in a repository',
+        input_schema=Schemas.create_issue_parameters
+    )
+    def create_issue(self, parameters):
+        owner, repo = self.state.get("owner"), parameters["repo_name"]
+        data = {"title": parameters["title"], "body": parameters["body"]}
+        return self.github_request(method="post", route=f"/repos/{owner}/{repo}/issues", data=data)
```

Comment:
Added back this action that was commented out in the old PR, because the process uses it like `self.plugin_inst.create_issue(...)`